packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -0,0 +1,247 @@
+import {Component, h, State, Prop, Host} from '@stencil/core';
+import {
+  Bindings,
+  InitializableComponent,
+  BindStateToController,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import {
+  BreadcrumbManagerState,
+  BreadcrumbManager,
+  buildBreadcrumbManager,
+  FacetManager,
+  FacetManagerState,
+  buildFacetManager,
+} from '@coveo/headless';
+import {Button} from '../common/button';
+import CloseIcon from 'coveo-styleguide/resources/icons/svg/close.svg';
+import {getFieldValueCaption} from '../../utils/field-utils';
+
+interface Breadcrumb {
+  facetId: string;
+  label: string;
+  formattedValue: string | string[];
+  deselect: () => void;
+}
+
+const SEPARATOR = ' / ';
+const ELLIPSIS = '...';
+
+/**
+ * The `atomic-breadbox` component creates breadcrumbs that display a summary of the currently active facet values.
+ *
+ * @part breadcrumb-button - A single breadcrumb button.
+ * @part breadcrumb-label - The breadcrumb label, associated with the facet.
+ * @part breadcrumb-value - The breadcrumb formatted value.
+ * @part show-more - The button to display all breadcrumbs.
+ * @part show-less - The button to display less breadcrumbs.
+ * @part label - The "Filters" label.
+ * @part clear - The button to clear all filters.
+ */
+@Component({
+  tag: 'atomic-breadbox',
+  styleUrl: 'atomic-breadbox.pcss',
+  shadow: true,
+})
+export class AtomicBreadbox implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  private breadcrumbManager!: BreadcrumbManager;
+  facetManager!: FacetManager;
+
+  @BindStateToController('breadcrumbManager')
+  @State()
+  private breadcrumbManagerState!: BreadcrumbManagerState;
+  @BindStateToController('facetManager')
+  @State()
+  public facetManagerState!: FacetManagerState;
+  @State() public error!: Error;
+  @State() private isCollapsed = true;
+
+  /**
+   * Number of breadcrumbs to display when collapsed.
+   */
+  @Prop() public collapseThreshold = 5;
```

Comment:
@btaillon + @johnnyCoveo : I guess the same UX principle that was done for multi-text-value should be applied here. Which means that no matter what, we should never be displaying `and one more`.